tests/makefiles/style.css

```
@@ -0,0 +1,542 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+:root {
+    --bg-color: white;
+    --fg-color: #505050;
```

Great work on the stylesheet in general! However, I'm going to be a bit of a bore and suggest that for the body text of a document this long, it seems more reader-accommodating to go with higher contrast:
```suggestion
    --fg-color: #000000;
```
(Compare with e.g., [JSON:API](https://jsonapi.org/format/), [Semantic Versioning](https://semver.org/). To compensate the headlines could be set in grey or a color.)